mscore/propertymenu.cpp

```diff
@@ -522,12 +524,21 @@ void ScoreView::elementPropertyAction(const QString& cmd, Element* e)
 //            editFretDiagram(static_cast<FretDiagram*>(e));
       else if (cmd == "staff-props") {
             Fraction tick = {-1,1};
```

Comment:
In principle, we'd also hit that test for tick -1 if any elements other than chordrests, notes, measures, or instrument names are capable of displaying staff properties in their context menu.